app/helpers/earthworks_blacklight_helper.rb

```diff
@@ -8,6 +8,14 @@ def show_sidebar_classes
     'page-sidebar col-lg-2'
   end
 
+  def main_content_classes
```

Comment:
Would you mind dropping comments on each method that states this is a Blacklight override.